### PR TITLE
update quickstart.md

### PR DESCRIPTION
**Pull request by simonLeary42**
what's with the __ in the title?
I ran into a problem installing on a blank Ubuntu 18.04.6 LTS, I needed python3-venv and I had to delete files and restart
Answered a question I had
fixed a typo
Answered another question I had

### CODE REVIEW

File: docs/setup-guides/production-baremetal/quickstart.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Baremetal QuickStart
 
-### Install Dependencies on the controller \_\_ node
```
Comment (diurnalist) [inline docs/setup-guides/production-baremetal/quickstart.md:3]: Huh, not sure why that was there!

File: docs/setup-guides/production-baremetal/quickstart.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Commands in this section will use the system package manager, and run with root
 
 ```
 sudo apt update && sudo apt install -f -y \
-    python3 python3-virtualenv virtualenv \
+    python3 python3-virtualenv python3-venv virtualenv \
```
Comment (diurnalist) [inline docs/setup-guides/production-baremetal/quickstart.md:13]: I think you're right that we should use `python3-venv`. I think `python3-virtualenv` should actually be removed from here! In a Python 3 context `venv` is used as part of stdlib, but I guess Ubuntu doesn't ship that as part of the base python3 package for space reasons (?).

File: docs/setup-guides/production-baremetal/quickstart.md [hunk 3]
```diff
@@ -168,7 +168,7 @@ If you are only using two interfaces, you must do some additional configuration
         # This should be your public IP block assigned to your deployment.
         cidr: 0.0.0.0/32
     ```
-5.  To enable bare metal provisioning, provide a vlan and subnet for ironic to use.
+5.  To enable bare metal provisioning, provide a vlan and subnet for ironic to use. This vlan should be included in the neutron on_demand_vlan_range above.
```
Comment (diurnalist) [inline docs/setup-guides/production-baremetal/quickstart.md]:
```suggestion
5.  To enable bare metal provisioning, provide a vlan and subnet for ironic to use. This vlan should be included in the neutron `on_demand_vlan_range` above.
```

**Review by diurnalist — APPROVED** [review]
Thanks!

**Review by msherman64 — APPROVED** [review]

**Pull request merged**